kpi/serializers/v2/asset_permission_assignment.py

```diff
@@ -52,6 +52,11 @@ def create(self, validated_data):
                 'user': "Owner's permissions cannot be assigned explicitly"})
         permission = validated_data['permission']
         partial_permissions = validated_data.get('partial_permissions', None)
+
+        bulk = self.context.get('bulk', False)
+        if bulk is False and asset.has_deployment:
+            asset.deployment.remove_from_flag(user_id=user.pk)
```

Comment:
I guess there's another place where the flag is removed for bulk assignments (verify)

Reply:
OK, it's handled by `AssetPermissionAssignmentViewSet.bulk_assignments()`. May I request a comment here mentioning that :)